packages/component-base/index.d.ts

```diff
@@ -1,5 +1,6 @@
 export { ActiveMixin } from './src/active-mixin.js';
 export { DisabledMixin } from './src/disabled-mixin.js';
+export { ElementMixin } from './src/vaadin-element-mixin.js';
```

Comment:
Let's change this file name to be `element-mixin.js` and not use `vaadin-` prefix.

Same applies to `dir-mixin.js` - please also add it here BTW.